docs/application/dotnet/guides/multimedia/audio.md

```diff
@@ -140,8 +140,34 @@ To query audio device information:
         > **Note**   
 		> The initial running state of the connected device is `false`, which means the connected device is not running.
```

Comment:
Just an observation that the Note tag is not properly aligned and not following MD style. Kindly fix the note tag. Also, suggesting to rephrase as:

>**Note**
>
> The initial running state of the connected device is `false`. This means that the connected device is not running.

Kindly verify the technical meaning before incorporating suggested rephrase.